RecoHI/HiJetAlgos/src/VoronoiAlgorithm.cc

```diff
@@ -174,6 +174,8 @@ namespace {
 
 	class problem_t {
 	public:
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-const-variable"
```

Comment:
GCC ignores other possible compilers. We can `clang` -> `GCC` and should do the trick. Will check it on Monday.